01_prepare_host.sh

```diff
@@ -60,7 +60,7 @@ for IMAGE_VAR in IRONIC_IMAGE IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE
     sudo "${CONTAINER_RUNTIME}" pull "$IMAGE"
 done
 
-# Download IPA and CentOS 7 Images
+# Download IPA and CentOS 8 Images
```

Comment:
I don't think we care about testing deployment of centos8 in this PR, instead we need to set e.g IRONIC_LOCAL_IMAGE to point to your fork:

https://github.com/metal3-io/metal3-dev-env/blob/master/02_configure_host.sh#L135